src/tiled/propertybrowser.cpp

```diff
@@ -886,14 +886,14 @@ void PropertyBrowser::addTileProperties()
     probabilityProperty->setAttribute(QLatin1String("decimals"), 3);
     probabilityProperty->setToolTip(tr("Relative chance this tile will be picked"));
     probabilityProperty->setEnabled(mTilesetDocument);
-    QtVariantProperty *overrideProperty = addProperty(OverrideTransformationFlagsProperty, QVariant::Bool, tr("Override Flipping"), groupProperty);
-    QtVariantProperty *flipHorizontallyProperty = addProperty(AllowFlipHorizontallyProperty, QVariant::Bool, tr("Flip Horizontally"), groupProperty);
-    QtVariantProperty *flipVerticallyProperty = addProperty(AllowFlipVerticallyProperty, QVariant::Bool, tr("Flip Vertically"), groupProperty);
-    QtVariantProperty *rotateProperty = addProperty(AllowRotateProperty, QVariant::Bool, tr("Rotate"), groupProperty);
-    flipHorizontallyProperty->setEnabled(false);
-    flipVerticallyProperty->setEnabled(false);
-    rotateProperty->setEnabled(false);
-    overrideProperty->setEnabled(mTilesetDocument);
+//    QtVariantProperty *overrideProperty = addProperty(OverrideTransformationFlagsProperty, QVariant::Bool, tr("Override Flipping"), groupProperty);
+//    QtVariantProperty *flipHorizontallyProperty = addProperty(AllowFlipHorizontallyProperty, QVariant::Bool, tr("Flip Horizontally"), groupProperty);
+//    QtVariantProperty *flipVerticallyProperty = addProperty(AllowFlipVerticallyProperty, QVariant::Bool, tr("Flip Vertically"), groupProperty);
+//    QtVariantProperty *rotateProperty = addProperty(AllowRotateProperty, QVariant::Bool, tr("Rotate"), groupProperty);
+//    flipHorizontallyProperty->setEnabled(false);
+//    flipVerticallyProperty->setEnabled(false);
+//    rotateProperty->setEnabled(false);
+//    overrideProperty->setEnabled(mTilesetDocument);
```

Comment:
Should be removed entirely rather than commented out. Would be good to do this in a separate commit, so it can be reverted as a start on implementing it.